### DISCUSSION TITLE
About email address for submission

### DISCUSSION THREAD

**Discussion by KyungwanHan** [chulminy/CIVE497-CIVE700#152] [Q&A] [answered]
Hello, 

I just submitted the assignment file. Is uw.task@gmail.com the correct address? I'm asking because it's a Gmail address, not a university email.

Thank you.

**Comment by chulminy** [comment, marked as answer]
Yes. It's correct. I would manage the submission in a different email address. 
Since I map the school email to my gmail, I prefer to gmail. 

Chul Min

**Reply by KyungwanHan** [reply]
Thank you for the answer!
